app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -453,7 +447,7 @@ open class BrowserActivity : DuckDuckGoActivity() {
         }
         dialog.setOnShowListener { currentTab?.onFireDialogVisibilityChanged(isVisible = true) }
         dialog.setOnCancelListener {
-            pixel.fire(if (dialog.ctaVisible) FIRE_DIALOG_PROMOTED_CANCEL else FIRE_DIALOG_CANCEL)
```

Comment:
Do we still need to keep the `FIRE_DIALOG_PROMOTED_CANCEL` constant in the code? I see it's not used anymore.

Reply:
Good catch! I'll remove it